docs/topics/wasm/wasm-debugging.md

```diff
@@ -0,0 +1,146 @@
+   ```
+   >Alternatively, you can load **compose-example** by opening it directly from the file explorer in **File | Open**.
+   >
+   {type="note"}
+
+4. Once the application starts, open the following URL in your browser:
+
+   ```bash
+   http://localhost:8080/
+   ```
+   
+   > The port number can vary. If the 8080 port is unavailable, you can find the corresponding port number printed in the console
+   > after building the application.
+   >
+   {type="tip"}
+
+   You see a **Hello World!** button. Click it:
+
+   ![Click hello world](wasm-composeapp-browser-hello.png){width=650}
+
+   You see the Compose Multiplatform logo:
+
+   ![Compose app in browser](wasm-composeapp-browser.png){width=650}
+
+## Debug in the browser
+
+> Currently, debugging is only available in the browser. Debugging in 
+> [IntelliJ IDEA](https://youtrack.jetbrains.com/issue/KT-64683/Kotlin-Wasm-debugging-in-IntelliJ-IDEA) and 
+> [Fleet](https://youtrack.jetbrains.com/issue/KT-64684) is in the plan. 
+>
+{type="note"}
+
+You can debug this Compose Multiplatform application and the rest of the [Kotlin/Wasm examples](https://github.com/Kotlin/kotlin-wasm-examples/tree/main/compose-example)
+in your browser out of the box without additional configurations. However, for other projects, you may need to set the required configuration in the Gradle 
+build file. For more information, see [Configuration for debugging](#configuration-for-debugging).
+
+For this tutorial, we are using Chrome. If you use a different browser, the experience should be similar and the following 
+steps should also work. 
+
+To debug a Kotlin/Wasm application:
+
+1. In the browser window of the application, access developer tools by right-clicking and selecting the **Inspect** action.
+Alternatively, you can use either the **Fn + F12** or just the  **F12** shortcut.
+
+2. Switch to the **Sources** tab and find the Kotlin file to debug.
+
+3. Set breakpoints in the Kotlin file by clicking on the numbers of the code lines you want to inspect. Only the code lines 
+with darker numbers accept breakpoints.
+
+![Set breakpoints](wasm-breakpoints.png){width=700}
+
+4. Interact with the application by clicking on the **Hello World!** button. This action triggers the execution of the 
+code, and the debugger pauses at each breakpoint.
+
+5. Inspect variables and code execution at the breakpoints by using the debugging control buttons (step over, step into, step out, and more).
+
+![Debug controls](wasm-debug-controls.png){width=700}
+
+6. Check the **Call stack** and **Scope** tool windows to trace the sequence of function calls and pinpoint the location of any errors.
+
+![Check call stack](wasm-debug-scope.png){width=700}
+
+7. Make the required changes to the code and [run the application](#run-the-application) again.
+
+### Configuration for debugging
+
+In projects requiring additional debugging configuration, you need to allow Webpack servers to access your sources. 
```

Comment:
By default, browsers don't have access to your project sources required for debugging. You can configure Webpack DevServer to serve the sources by adding the following code snippet to the Gradle build file: